docs/api_reference/python-api-reference.md

```diff
@@ -6,7 +6,11 @@ SPDX-License-Identifier: MPL-2.0
 
 # power_grid_model (Python API)
 
-This is the Python API reference for the `power-grid-model` library
+This is the Python API reference for the `power-grid-model` library.
+Due to the nature of how Python module works, we cannnot hide the implementation detail completely from the user.
+As a general rule, any Python modules/functions/classes which are not documented in this API documentation, 
+are internal implementations.
```

Comment:
```suggestion
```suggestion
Due to the nature of Python modules, we cannot hide implementation details completely from the user.
As a general rule, any Python module/function/class not documented in this API documentation
is considered internal implementation.
```

Reply:
The suggestion block stuff does not work well with readthedocs. Readthedocs treats them as separate code block and render them as is (no line wrapping etc.) instead treating them as text.